libs/@local/status/LICENSE-MIT.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright © 2022–, HASH
```

Comment:
```suggestion
Copyright © 2023–, HASH
```

Reply:
This is the same as all other LICENSE-MIT files within the repo, if we want to update the year I suggest we do it in a batch in a follow-up

Reply:
At least for `sarif` we have `2023--` instead. @nonparibus to decide

Reply:
The license year should begin the year the package/directory was created in. It's not very important, though.